schemas/test_cases/v0/misc.yaml

```diff
@@ -162,3 +162,23 @@
     enabled: true
     begin_on_batch: 100
     end_after_batch: 1
+
+- name: azure is invalid when both connection_string and credential specified
+  sanity_errors:
+    http://determined.ai/schemas/expconf/v0/azure.json:
+      - "credential and connection_string must not both be set"
+  case:
+    type: azure
+    container: container1
+    connection_string: conn_str
+    credential: my_pwd
+
+- name: azure is valid when both account_url and credential specified
+  complete_as:
+    - http://determined.ai/schemas/expconf/v0/azure.json
+  case:
+    type: azure
+    container: my_container
+    account_url: my_acc_url
+    credential: my_pwd
+    
```

Comment:
github is indicating something weird in this file, again probably whitespace related.